src/main/java/seedu/bankwithus/BankWithUs.java

```diff
@@ -83,4 +84,14 @@ public static void main(String[] args) {
             return;
         }
     }
+
+    public  void createAccount() {
+        System.out.println("Whats your name?");
+        ui.createScanner();
+        String userName = ui.getNextLine();
```

Comment:
With the above suggestion in mind, it might be preferable to extract out sections as functions, keeping in mind SLAP. A suggestion can be to extract line 89 to 91as readUserName(), and line 92 to 93 as readInitialBalance().

Reply:
will extract this segment away into UI class